benefits/eligibility/views.py

```diff
@@ -40,21 +40,20 @@ def index(request):
     help_page = reverse(ROUTE_HELP)
 
     page = viewmodels.Page(
-        classes="offset-lg-1 col-lg-9",
         title=_("eligibility.pages.index.title"),
         headline=_("eligibility.pages.index.headline"),
         paragraphs=[
             format_html(_("eligibility.pages.index.p[0]%(info_link)s") % {"info_link": f"{help_page}#what-is-cal-itp"})
         ],
-        forms=forms.EligibilityVerifierSelectionForm(agency=agency),
+        forms=forms.EligibilityVerifierSelectionForm(agency=agency, classes="offset-lg-1 col-lg-9"),
     )
 
     ctx = page.context_dict()
     ctx["help_page"] = help_page
     ctx["show_help_text"] = True
 
     if request.method == "POST":
-        form = forms.EligibilityVerifierSelectionForm(data=request.POST, agency=agency)
+        form = forms.EligibilityVerifierSelectionForm(data=request.POST, agency=agency, classes="offset-lg-1 col-lg-9")
```

Comment:
I wasn't sure if the class attribute should be set for this `form` and the one on line 188 too? I did it for all of them cos the page wouldn't lose otherwise.

Reply:
Yes, for the `POST` back case, if the form is invalid (which it shouldn't be... but if...) then we would send the user back to the form page to try again, so you need it here.

And then definitely need it in the `GET` since that is the initial page load of the form.

[See my comment above](https://github.com/cal-itp/benefits/pull/1023#discussion_r990225774), maybe a simplification?